examples/3d/irradiance_volumes.rs

```diff
@@ -0,0 +1,446 @@
+//! This example shows how irradiance volumes affect the indirect lighting of
+//! objects in a scene.
+//!
+//! The controls are as follows:
+//!
+//! * Space toggles the irradiance volume on and off.
+//!
+//! * Enter toggles the camera rotation on and off.
+//!
+//! * Tab switches the object between a plain sphere and a running fox.
+//!
+//! * Clicking anywhere moves the object.
+
+use bevy::math::vec3;
+use bevy::pbr::irradiance_volume::IrradianceVolume;
+use bevy::prelude::shape::UVSphere;
+use bevy::prelude::*;
+use bevy::window::PrimaryWindow;
+
+// Rotation speed in radians per frame.
+const ROTATION_SPEED: f32 = 0.005;
+
+const FOX_SCALE: f32 = 0.05;
+const SPHERE_SCALE: f32 = 2.0;
+
+const IRRADIANCE_VOLUME_INTENSITY: f32 = 150.0;
+
+static DISABLE_IRRADIANCE_VOLUME_HELP_TEXT: &str = "Press Space to disable the irradiance volume";
+static ENABLE_IRRADIANCE_VOLUME_HELP_TEXT: &str = "Press Space to enable the irradiance volume";
+
+static STOP_ROTATION_HELP_TEXT: &str = "Press Enter to stop rotation";
```

Comment:
Can you format the text to be like `(Space): [Enable/Disable] he irradiance volume`